docs/FAQ.md

```diff
@@ -8,6 +8,12 @@ this document. Few of answers assumes that the MCM is being used in conjuction w
 <!--- TOC BEGIN -->
 * [Basics](#basics)
   * [What is Machine Controller Manager?](#what-is-machine-controller-manager)
+  * [What are different sub-controllers in MCM ?](#What-are-different-sub-controllers-in-MCM)
+  * [What is safety-controller in MCM ?](#What-is-safety-controller-in-MCM)
+  * [Why is my machine deleted ?](#Why-is-my-machine-deleted)
+
+* [How to?](#how-to)
+  * [How to install the MCM in a Kubernetes cluster?](#How-to-install-the-MCM-in-a-Kubernetes-cluster)
```

Comment:
Changed `kubernetes` to 'Kubernetes` everywhere so as to respect the proper noun.